### PR TITLE
fix: postinstall and create-xplat-workspace

### PR DESCRIPTION
**Pull request by m-abs**
* create-xplat-workspace didn't add all the genarated files
* postinstall failed to patch the nx-formatter in latest release

### CODE REVIEW

File: scripts/postinstall.ts [hunk 1]
```diff
@@ -31,26 +35,18 @@ export function fixFormatter() {
...
-  // Old patch line, globs should've been wrapped in quotes.
-  const oldPatchLine = `    patterns.push("xplat\/**\/*.ts", "!apps/**/platforms/{android,ios}/**/*.ts");`;
-  const patchLine = `    patterns.push('\"xplat\/**\/*.ts\"', '\"!apps/**/platforms/{android,ios}/**/*.ts\"');`;
-  if (formatContent.indexOf(oldPatchLine) !== -1) {
-    console.log('Old patch for nx format have been applied, replace with new patch');
-    fs.writeFileSync(formatPath, formatContent.replace(oldPatchLine, patchLine));
-    return;
-  }
-
+  const patchLine = `    // PATCHED by @nstudio/xplat\n    patterns.push('"xplat/**/*.ts"', '"!apps/**/platforms/{android,ios}/**/*"');`;
```
Comment (NathanWalker) [inline scripts/postinstall.ts]: I had issues the last time this patch line was modified - can you confirm that this patch works properly in a fresh workspace based off latest Nx?

Thanks for this PR @m-abs

Comment (NathanWalker) [reply scripts/postinstall.ts]: Sounds like you did but just want to be extra sure 🤗

Comment (m-abs) [reply scripts/postinstall.ts]: I tested it on a fresh project.  But I'd really like an automated way of testing it.

Comment (NathanWalker) [reply scripts/postinstall.ts]: That would be nice :) thanks again I’ll merge and test this locally befor bumping a release 👍

**Comment by bubbleheadinc** [comment]
Is this fix in 6.2.9? I'm getting nx-formatter issue when installing xplat schematics in 6.2.9. 

```npm install @nstudio/schematics --save-dev

@nstudio/schematics@6.2.9 postinstall /dev/xplattest/node_modules/@nstudio/schematics
node scripts/postinstall.js

An issue were detected during patching the nx-formatter Error: Apply couldn't patch for nx format
    at fixFormatter (/dev/xplattest/node_modules/@nstudio/schematics/scripts/postinstall.js:46:15)
    at updateConfig (/dev/xplattest/node_modules/@nstudio/schematics/scripts/postinstall.js:28:9)
    at Object.<anonymous> (/dev/xplattest/node_modules/@nstudio/schematics/scripts/postinstall.js:35:1)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
    at Function.Module.runMain (module.js:684:10)
    at startup (bootstrap_node.js:187:16)
+ @nstudio/schematics@6.2.9
updated 1 package in 7.417s
```

**Pull request merged**